### PR TITLE
Fluid Update

### PR DESCRIPTION
**Pull request by OmegaHaxors**
Textures are broken, i'm hoping someone here can figure that out.
Adds the ability to add custom fluids to ELN.

### CODE REVIEW

File: src/main/java/mods/eln/Eln.java [hunk 1]
```diff
@@ -4737,6 +4805,14 @@ name, new ThermalLoadInitializerByPowerDrop(780, -100, 10, 2)
             );
             transparentNodeItem.addDescriptor(subId + (id << 6), desc);
         }
+        {
```
Comment (OmegaHaxors) [inline src/main/java/mods/eln/Eln.java:4808]: I don't know how this got in here, feel free to reject this change.

File: src/main/java/mods/eln/Eln.java [hunk 2]
```diff
@@ -6067,6 +6143,14 @@ private void recipePassiveComponent() {
             'C', findItemStack("Copper Thermal Cable"),
             'D', findItemStack("Small Passive Thermal Dissipator")
         );
+
```
Comment (OmegaHaxors) [inline src/main/java/mods/eln/Eln.java:6146]: Again, this shouldn't be here

File: src/main/java/mods/eln/transparentnode/ThermalHeatExchanger.kt [hunk 3]
```diff
@@ -183,6 +184,15 @@ class ThermalHeatExchangerElement(
             thermalPairs.add(ThermalPairing(ic2hotcoolant, ic2coolant, 640.0 / 7.0, 9, 1.0, false))
         }
 
+        if (ic2hotwater != null) {
```
Comment (OmegaHaxors) [inline src/main/java/mods/eln/transparentnode/ThermalHeatExchanger.kt:187]: This isn't supposed to be here either but it's a nice little feature.
Adds the ability to convert IC2 hotspring water into normal water and gain a little heat in the process.

File: src/main/java/mods/eln/fluid/ElnItemBlock.java [hunk 4]
```diff
@@ -0,0 +1,12 @@
+package mods.eln.fluid;
```
Comment (jrddunbr) [inline src/main/java/mods/eln/fluid/ElnItemBlock.java:1]: Nit: This can be Kotlin.

**Review by jrddunbr — APPROVED** [review]
LGTM

**Pull request merged**